textdescriptives/components/utils.py

```diff
@@ -0,0 +1,60 @@
+"""Utility functions for calculating various text descriptives"""
+from spacy.tokens import Doc
+from spacy.tokens import Token
+from spacy.language import Language
+from pyphen import Pyphen
+
+
+@Language.factory("utilities")
```

Comment:
I would actually just make an utils script and then add these components in the other pipelines.

Then when setting the component you can check if it already there, if not then set it. That also solves the lack of dependencies.

Reply:
done in 1cf1527